include/bm/bm_sim/P4Objects.h

```diff
@@ -59,6 +59,8 @@ class Value;
 
 namespace bm {
 
+using std::string;
```

Comment:
I'd personally recommend against doing that, especially in a header file, even if namespaced

Reply:
no worries, you just need to fully qualify `std::string` (instead of `string`). You have 2 occurrences in the patch (`table_name` parameter in `check_next_nodes` function).

Reply:
removed in commit 4